Ginger/Ginger/ALM/ALMIntegration.cs

```diff
@@ -41,19 +41,21 @@ public class ALMIntegration
 
         private ALMIntegration()
         {
-            UpdateALMType((eALMType) WorkSpace.Instance.Solution.AlmType);
+            //UpdateALMType(ALMCore.DefaultAlmConfig.AlmType);
```

Comment:
Why this is commented? if no longer required then UpdateALMType function is also not required, it has 0 references